ts/features/services/search/screens/SearchScreen.tsx

```diff
@@ -211,7 +210,6 @@ export const SearchScreen = () => {
           paddingHorizontal: IOVisualCostants.appMarginDefault
         }}
         data={data?.institutions}
-        estimatedItemSize={LIST_ITEM_HEIGHT}
```

Comment:
Question: is it necessary to remove this prop?

Reply:
The removal was needed while trying to upgrade to flashlist v2, since we can't upgrade the major yet we can restore the property